handbook/engineering/web/index.md

```diff
@@ -13,7 +13,20 @@ The web team's current focus is documented in [the tracking issue for the curren
 
 ## Goals
 
-TODO
+### Long term
+
+***Deliver our users the full, unique value of [extensions](https://docs.sourcegraph.com/extensions).***
```

Comment:
Sentence nitpick: I think it should be either

```suggestion
***Deliver to our users the full, unique value of [extensions](https://docs.sourcegraph.com/extensions).***
```

or

```suggestion
***Deliver the full, unique value of [extensions](https://docs.sourcegraph.com/extensions) to our users.***
```

Reply:
I prefer the second of these suggestions.